### PR TITLE
Set embedded cluster version Helm value for the adminconsole addon

### PR DESCRIPTION
**Pull request by sgalsaleh**

### CODE REVIEW

File: pkg/addons/adminconsole/adminconsole.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ var (
 )
 
 // protectedFields are helm values that are not overwritten when upgrading the addon.
-var protectedFields = []string{"automation", "embeddedClusterID"}
+var protectedFields = []string{"automation", "embeddedClusterID", "embeddedClusterVersion"}
```
Comment (laverya) [inline pkg/addons/adminconsole/adminconsole.go:42]: this should not be protected, doing so will prevent it from changing when we update embedded-cluster

File: pkg/addons/adminconsole/adminconsole.go [hunk 2]
```diff
@@ -53,7 +53,8 @@ var helmValues = map[string]interface{}{
 		"enabled":  true,
 		"nodePort": DEFAULT_ADMIN_CONSOLE_NODE_PORT,
 	},
-	"embeddedClusterID": metrics.ClusterID().String(),
+	"embeddedClusterID":      metrics.ClusterID().String(),
+	"embeddedClusterVersion": defaults.Version,
```
Comment (laverya) [inline pkg/addons/adminconsole/adminconsole.go:57]: IIRC this isn't actually used for anything besides populating a configmap?

Comment (laverya) [reply pkg/addons/adminconsole/adminconsole.go:57]: Is that configmap actually used for something?

Comment (sgalsaleh) [reply pkg/addons/adminconsole/adminconsole.go:57]: it's passed as an environment variable to the kotsadm deployment for reporting purposes

File: pkg/addons/adminconsole/adminconsole.go (hunk 1 above)
Comment (sgalsaleh) [reply pkg/addons/adminconsole/adminconsole.go:42]: whoops. good catch.

File: pkg/addons/adminconsole/adminconsole.go (hunk 2 above)
Comment (sgalsaleh) [reply pkg/addons/adminconsole/adminconsole.go:57]: https://github.com/replicatedhq/kots-helm/pull/35

Comment (sgalsaleh) [reply pkg/addons/adminconsole/adminconsole.go:57]: alternatively, we can store the embedded cluster in the cluster somewhere and have KOTS detect it from there. not sure which approach is better.

**Review by laverya — APPROVED** [review]

File: pkg/addons/adminconsole/adminconsole.go (hunk 2 above)
Comment (laverya) [reply pkg/addons/adminconsole/adminconsole.go:57]: oh gotcha
yeah feel free to fix this then

**Pull request merged**